in_toto/in_toto_sign.py

```diff
@@ -207,33 +234,87 @@ def main():
   arguments it then dumps the corresponding file.
   """
   args = parse_args()
-  rsa_key = in_toto.util.import_rsa_key_from_file(args.key)
+  length = len(args.keys)
+  rsa_key = in_toto.util.import_rsa_key_from_file(args.keys[length-1])
```

Comment:
This comes a little unexpected. I wonder if it wouldn't be better to separate `sign` and `verify` after all, e.g. using a subparser as suggested earlier.